docs/data-formats.md

```diff
@@ -166,6 +182,24 @@ The `required` attribute specifies that if the type is not marked, during the ma
 
 ## Substitution Format
 
+Substitutions direct the linker to replace specific method's body with either a throw or return constant statements.
+
+Substitutions only work for methods from assemblies which are linked with assembly action `link`, in all other cases substitutions are ignored (no warning or message is issued).
```

Comment:
We don't output the substituted IL, but couldn't the substitutions still affect constant propagation?

Reply:
You're right - I didn't even realize that. Changed the comment. It's relatively unlikely, would basically require a trimmed assembly to call non-trimmed on, but it's is a possible case.